internal/drivers/manager.go

```diff
@@ -414,7 +414,7 @@ func (m *Manager) CleanPools(ctx context.Context, destroyBusy, destroyFree bool)
 	for _, pool := range m.poolMap {
 		err := m.cleanPool(ctx, pool, destroyBusy, destroyFree)
 		if err != nil {
-			return err
+			logrus.Errorf("failed to clean pool %s with error: %s", pool, err)
```

Comment:
probably still return the error on L421 instead of returning nil. But continue to try to clean all the pools

Reply:
done